src/ios/FirebaseMessagingPlugin.m

```diff
@@ -36,6 +36,14 @@ - (void)requestPermission:(CDVInvokedUrlCommand *)command {
     [[UIApplication sharedApplication] registerForRemoteNotifications];
 }
 
+- (void)clearMessages:(CDVInvokedUrlCommand *)command {
+    UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
+    [center removeAllDeliveredNotifications];
+    [center removeAllPendingNotificationRequests];
```

Comment:
Why do you remove pending notifications here?

Reply:
No reason 😢  Like I said, I have 0 experience with Objective C iOS development, I come from Cordova world. We had to migrate our project from `phonegap-push` to your plugin due to incompatibilities with latest `cordova-ios@6.1.0`. If you say I should not remove pending notifications here - I will gladly fix that :)

Reply:
Better to remove this line. Currently it’s not possible to create scheduled notifications via FCM. Having removeAllDeliveredNotifications should be enough.

Reply:
Thank you, I have commited the changes, see last commit to this PR.